start.py

```diff
@@ -28,12 +28,13 @@ def main():
 
 
 def load_implementations():
-    d = get_step_impl_dir()
-    if path.exists(d):
-        load_files(d)
-    else:
-        logging.error(
-            'can not load implementations from {}. {} does not exist.'.format(d, d))
+    d = get_step_impl_dirs()
+    for impl_dir in d:
+        if not path.exists(impl_dir):
+            logging.error('can not load implementations from {}. {} does not exist.'.format(impl_dir, impl_dir))
+            return
```

Comment:
Should we stop scanning other `dir` if one does not exist? Previously there was only one dir so it was ok to `exit`.